fractal-puppy/Engine
# Feature/enemy update

With this PR new enemies will be able to be added by dragging the prefab and they will be fully operational just like that.

To test:

Open the graveyard scene
Add one of each prefab
- BuriedEnemy
- FlyingSkull
- Bandolero

try using play and check that they work in the scene.
Beware that due to a bug with the hitboxes some exceptions unrelated to this PR may appear.

If they are not moving check that the GO with the script is touching the navmesh.
